ui/README.md

```diff
@@ -51,7 +51,13 @@ To run the UI localy follow the steps:
 
 Get them from the project settings on the firebase dashboard. Read [this article](https://support.google.com/firebase/answer/7015592?hl=en#zippy=%2Cin-this-article) to know how to get your porject config
 
-4. Start the local server running the app:
+4. To interact with the contract, you need to set the Goerli RPC. Set this variable on the .env file
+
+   ```bash
+   VITE_GOERLI_RPC
+   ```
+
+5. Start the local server running the app:
```

Comment:
tagging @csosamorales for this since I don't how to do that for ui.

Reply:
@jsonsivar what do you mean with contract address? the ABI?

Reply:
I mean the address of the contract (ie for Goerli 0x8795608346Eb475E42e69F1281008AEAa522479D). Somewhere we are setting that in the frontend right? I just mean how we set that should be clear and documented, and ideally some flag or env var or config.

Reply:
We are replacing the file `ui/src/integrations/ethereum/contracts/FleekERC721.json` with the file on contracts deployments folder. That's how we update the abi and the address. I think we could provide someway to get this json hosted somewhere, even if it is in github to provide a kind of env oriented selection for local/deployed versions. But this still a task to be done, we have [this](https://app.shortcut.com/fleekxyz/story/701/ui-create-env-variables-based-on-env-dev-prod) card that iirc is about that.

Reply:
Ah got it! That's totally fine for now, but we should still put a section in the README saying this.  I can make an edit. Just something like:

```
### Setting Contract Address and ABI

The contract address and ABI is set by pointing `ui/src/integrations/ethereum/contracts/FleekERC721.json` to the file from the deployment outputs in the contract sub project.

This can be a local deployment or a deployment on one of the networks.  This maintains consistency between the deployed contracts and the info in the UI configuration.
```

Also maybe one progression of that can be to make a `dev` or `prd` folder so like `ui/src/integrations/ethereum/contracts/dev/FleekERC721.json`. Another idea is to use a --network flag for UI build, and it will pull the latest json from the deployments folder that matches the network.  Issue with that is maybe we need it locked to a specific version so it doesn't break things if we have a new deployment but not yet ready to update UI to it.